app/src/main/java/org/piramalswasthya/sakhi/configuration/AdolescentHealthFormDataset.kt

```
@@ -0,0 +1,303 @@
+            ben.noOfPacketsDistributed = noOfPacketsDistributed.value?.toInt()
+            ben.place = place.value
+            ben.referredToHealthFacility = referredToHealthFacility.value
+            ben.counselingProvided = counselingProvided.value == counselingProvided.entries!![0]
+            ben.counselingType = counselingType.value
+            ben.followUpDate = getLongFromDate(followUpDate.value)
+            ben.referralStatus = referralStatus.value
+        }
+    }
+
+    fun setImageUriToFormElement(lastImageFormId: Int, dpUri: Uri) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)